playpen/mongoengine/validation_check.py

```diff
@@ -0,0 +1,153 @@
+#!/usr/bin/env python2
+
```

Comment:
A module docstring should be added which describes usage. The user needs to know this needs to be run as the `apache` user. Something like `sudo -u apache python validation_check.py`.